guides/common/modules/proc_running-installer.adoc

```diff
@@ -0,0 +1,17 @@
+[id="running-installer_{context}"]
```

Comment:
Please ensure that the anchor, title, and file name match.

Reply:
Our d/s modular docs reference says not to include the prefix in the anchor ID but I do not mind including it for the sake of consistency with the other modules. Will do.